### ISSUE TITLE
Are genes selectable?

### ISSUE THREAD

**Issue by AlistairNWard** [iobio/sv.iobio#130]
Hover over a gene in the genes track and the cursor changes to indicate that you can click on the gene. Clicking does nothing though. Are we intending to allow users to click and show some gene information? If not, we shouldn't imply you can click.

**Comment by emersonlebleu** [comment]
Yes I think we want to but I'm not certain exactly what information about a gene might be relevant aside from its HPO associations which we are already showing. 

But I think there are possible important things we could want to show about that gene no? I can take the hover effect off until we know as well?

**Comment by AlistairNWard** [comment]
HPO associations, OMIM information, PubMed information, links to other resources (e.g. uniprot, alphafold etc). There's lots of information on genes.

**Comment by emersonlebleu** [comment]
#91 This is essentially the same as that one but here is some additional information and thoughts. 

There is lots of information available, some of which we are using in other contexts though with the gene cards. Those HPO associations and OMIM/ORPHA associations are there when we look at the SV. We could maybe go in more depth, pubmed and other more protein related information could be interesting. I agree there are lots of interesting things we might be able to put here. One thing that also comes to mind is function or pathway information.

I'm reluctant to force the user to click on each gene to get the HPO & OMIM associations for them. I like that they can kind of review them at the SV level. But I could be convinced.

**Comment by AlistairNWard** [comment]
HPO associations that the proband has should be available without clicking on anything, but the rest of the associations for the gene could be a click away. Same with OMIM. There is a lot of information from all these resources so we need the important stuff easy to access and more details can be a click away.

**Comment by emersonlebleu** [comment]
Fixed this they are not clickable.
